libs/simdvec/build.gradle

```diff
@@ -23,6 +23,19 @@ dependencies {
   }
 }
 
+tasks.named("compileMain21Java").configure {
+  options.compilerArgs << '--add-modules=jdk.incubator.vector'
+  // we remove Werror, since incubating suppression (-Xlint:-incubating)
+  // is only support since JDK 22
+  options.compilerArgs -= '-Werror'
```

Comment:
@breskeby as discussed off-line, this small change disables warnings-as-error for this small sourceset. I think that the change is inline with what we discussed?